README.md

```diff
@@ -5,3 +5,327 @@
+
+```
+smartmeter-datacollector --config datacollector.ini
+```
+
+The following command line arguments are supported:
+* `-h, --help`: Shows the help output of `smartmeter-datacollector`.
+* `-c, --config CONFIG`: Path to the `.ini` configuration file.
+* `-d, --dev`: Enable development mode.
+
+### Configuration
+
+The configuration file can be located anywhere and use any filename. If no `.ini` configuration file is specified a default configuration with the following options is used:
+* Landys+Gir E450 smart meter in unencrypted mode connected to `/dev/ttyUSB0`
+* `LOGGER` sink
+* `MQTT` sink connected to a local broker without encryption or authentication
+
+## Run as a systemd service
+
+When `smartmeter-datacollector` has been installed as a Debian package it provides a systemd .service file named `python3-smartmeter-datacollector.service`. Therefore the service can be managed using the `systemctl` command:
+
+```
+sudo systemctl start|stop|restart python3-smartmeter-datacollector
+```
+
+Enabling or disabling the service can also be managed using the `systemctl` command:
+
+```
+sudo systemctl enable|disable python3-smartmeter-datacollector
+```
+
+Inspecting the log messages can be done using the `journalctl` command:
+
+```
+journalctl -u python3-smartmeter-datacollector [-f]
+```
+
+### Configuration
+
+The systemd service expectes the `.ini` configuration to be located at
+`/var/lib/smartmeter-datacollector/datacollector.ini`. This is also the default location used by the [`smartmeter-datacollector-configurator`](https://github.com/scs/smartmeter-datacollector-configurator) webinterface to load and save the managed configuration.
+
+# How to develop
+
+Help from the community for the `smartmeter-datacollector` project is always welcome. Please follow the next few chapters to setup a working development environment.
+
+## Development Requirements
+
+* Python >= 3.7
+* [`pipenv`](https://pipenv.pypa.io/en/latest/)
+* Optional software packages (Debian / Ubuntu)
+  * python3-all
+  * debhelper
+  * dh-python
+  * dh-systemd
+
+## Installation
+
+### Debian / Ubuntu
+
+To install the listed minimal requirements run the following command:
+
+```
+sudo apt install git python3 pipenv
+```
+
+To install the optional requirements also run the following command:
+
+```
+sudo apt install python3-all debhelper dh-python dh-systemd
+```
+
+## Checkout the code
+
+Use `git` to clone / checkout `smartmeter-datacollector` from GitHub using
+
+```
+git clone https://github.com/scs/smartmeter-datacollector.git
+```
+
+## Setup Development Environment
+
+`smartmeter-datacollector` uses [`pipenv`](https://pipenv.pypa.io/en/latest/) to manage its dependencies and setup a virtual environment. Run the following command to setup the initial development environment:
+
+```
+pipenv install --dev
+```
+
+This will install all runtime and development dependencies for `smartmeter-datacollector`. Now you are ready to start working on `smartmeter-datacollector`.
```

Comment:
Detail/for clarification: This will install all runtime and development dependencies for `smartmeter-datacollector` in a new virtual environment.

Reply:
added